### ISSUE TITLE
no one else cares about "context aware"

### ISSUE THREAD

**Issue by timm** [timm/700timesfaster#12]
context aware is my thing. no one else cares about it. find all references to "context aware" and edit it out

eg

"Our proposed method is to build local models, in order to makelearners context aware and decrease training time. To do this, wefirst cluster our data. Then we build a model for each cluster, paral-lelly. This process is shown in Figure 4."

**Comment by timm** [comment]
start of 3.2, need t line or two motivating the method

e.g. tuning a learner can be a complex task. In this section, we check if dividing up the data prior to tuning reduces the search space enough to seed up the learning.

**Comment by Suvodeep90** [comment]
removed the context aware parts and updated the section 3.2 with - "Training a models and hyper-parameter tuning technique can take large amount of time depending on the size of the training dataset. This makes the cross validation and retraining harder to achieve. We check if dividing up the data into small clusters and then train and tune models in parallel can speedup learning so our proposed method is to build local models, in order to reduce learner's training time. To do this, we first cluster our data. Then we build a model for each cluster, parallelly. This process is shown in Figure~\ref{fig:our_model}."
